components/org.wso2.carbon.identity.piicontroller/src/main/java/org/wso2/carbon/identity/piicontroller/ConsentConstants.java

```diff
@@ -35,7 +35,7 @@
 
     public static final String PII_CONTROLLER_CONNECTOR_NAME = "pii-controller";
     public static final String PII_CONTROLLER_CONNECTOR_FAMILY_NAME = "Consent Information Controller";
-    public static final String PII_CONTROLLER_CONNECTOR_CATEGORY = "Consent Management";
+    public static final String PII_CONTROLLER_CONNECTOR_CATEGORY = "Other Settings";
```

Comment:
As described in [1]  it has been decided to make UI component for governance connectors more generic. Thereby consent management configurations will fall under the `Other Settings` section.
```
Other Settings >
              Consent Information Controller
```
 I will introduce a new variable to avoid any confusion in the code.

[1] https://github.com/wso2/product-is/issues/8730

Reply:
introduced a new constant as `CONNECTORY_CATEGORY` with commit: 2a15877b7243a42335c80c7ba594f967116f16fa